### PR TITLE
[py] Moved Rust binary settings to `pyproject.toml` from `setup.py`

### PR DESCRIPTION
**Pull request by sandeepsuryaprasad**
### **User description**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Moved Rust binary settings to `pyproject.toml`  from `setup.py` thus completely eliminating the need of `setup.py`

### Motivation and Context
- This PR completely eliminates the need of `setup.py` as all the settings are moved to `pyproject.toml`
-  After the change is made, I ran the build on my local machine `bazel build //py:selenium-wheel` which is successful.
-  Installed `selenium` from `.whl` file, the installation was successful.
- Verified that `selenium-manager` is present in `selenium.webdriver.common` folder (in /linux, /macos, /windows)
- Verified the installation with `--no-binary` option as well.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

___

### **PR Type**
enhancement

___

### **Description**
- Removed the `setup.py` file, migrating its configurations to `pyproject.toml`.
- Updated `pyproject.toml` to include Rust binary settings for `selenium-manager`.
- Modified Bazel build configuration by removing `setup.py` from the source files.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>setup.py</strong><dd><code>Remove setup.py and migrate configurations</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

py/setup.py

<li>Removed the <code>setup.py</code> file entirely.<br> <li> Eliminated the need for <code>setup.py</code> by moving configurations.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14837/files#diff-722aeeec2821793eb6adf0c529cc7439c4b27ce78937cbb8840e94d4fc1c4017">+0/-38</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>pyproject.toml</strong><dd><code>Add Rust binary settings to pyproject.toml</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

py/pyproject.toml

- Added Rust binary settings for `selenium-manager`.

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14837/files#diff-3adb340b5a499e26b04507d972c31f5cf6fc27a6d81e3f7b547bf50e90c43be3">+3/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>BUILD.bazel</strong><dd><code>Update Bazel build configuration for setup.py removal</code>&nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

py/BUILD.bazel

- Removed `setup.py` from the list of source files.

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/selenium/pull/14837/files#diff-3e8ff3a52e8e6c0b195ea328e17c50ba6d90abca1ce1624110607da20691d7a9">+0/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by sandeepsuryaprasad** [comment]
@VietND96 Can you please build this one for me from CI and upload to test PyPI. So that i will verify the binaries..

**Comment by VietND96** [comment]
I think other script need to be updated, since publish CI failed https://github.com/SeleniumHQ/selenium/actions/runs/12113072764/job/33767482214
Will support you once I have time.

**Comment by VietND96** [comment]
Dist package is https://test.pypi.org/project/selenium/4.28.0.202412041759/

**Review by VietND96 — APPROVED** [review]

File: py/pyproject.toml [hunk 1]
```diff
@@ -44,6 +44,9 @@ exclude = ["test*"]
 namespaces = false
 # include-package-data is `true` by default in pyproject.toml
 
+[[tool.setuptools-rust.bins]]
+target = "selenium.webdriver.common.selenium-manager"
```
Comment (jameshilliard) [inline py/pyproject.toml:48]: This should have been:
```python
[[tool.setuptools-rust.ext-modules]]
target = "selenium.webdriver.common.selenium-manager"
binding = "Exec"
```
Fixed in #16074.

**Pull request merged**